Tests/Tests/Core/Extensions/UIKit/UIViewControllerTests.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  UIViewControllerTests.swift
+//  PovioKit_Tests
+//
+//  Created by Ndriqim Nagavci on 27/10/2022.
+//  Copyright © 2022 Povio Inc. All rights reserved.
+//
+
+import XCTest
+import PovioKit
+import PovioKitUI
+
+class UIViewControllerTests: XCTestCase {
+  func test_createAllButtonTypes() {
```

Comment:
Please update the function name to match the pattern with the functions below: `test_x_doesY`